### DISCUSSION TITLE
Hollow Knight doesnt work on build 1246 but working on previous build 1237

### DISCUSSION THREAD

**Discussion by rosenimcute** [skyline-emu/skyline#1700] [General]
Hollow Knight doesnt work on build 1246 but working on previous build 1237

Loads intro after start black screen only

Snapdragon 855/adreno 640

![Screenshot_20220716-143034_Skyline](https://user-images.githubusercontent.com/104092390/179343459-86eef508-f308-4cee-a954-30fd99d5a0a0.jpg)

**Comment by MrAnty** [comment]
I've noticed games not booting on the newer build, plus games running at a lower frame rate. Build 1237 is currently the best build at the moment in my opinion.

**Reply by rosenimcute** [reply]
yeah, hoping they will fix it soon on the next update.
